src/Kunstmaan/MenuBundle/Repository/MenuItemRepository.php

```diff
@@ -25,7 +26,8 @@ public function getMenuItemsForLanguage($menuName, $locale)
             ->setParameter('locale', $locale)
             ->andWhere('m.name = :name')
             ->setParameter('name', $menuName)
-            ->andWhere('nt.online = 1');
+            ->andWhere('nt.online = 1 OR mi.type = :url_type')
+            ->setParameter('url_type', MenuItem::TYPE_URL_LINK);
```

Comment:
It is better to use `BaseMenuItem::TYPE_URL_LINK` because the MenuItem entity can be overridden via configuration.